### PR TITLE
Add option to delete search input

### PR DESCRIPTION
**Pull request by bliemelh**
Hi @junkerm ! I added an option to delete the search input in project perspective and would be grateful for feedback.
https://trello.com/c/MgbG1KVX/364-option-zum-l%C3%B6schen-der-suche

### CODE REVIEW

**Review by junkerm — CHANGES_REQUESTED** [review]
the function works! Could you remove the border around the "x"?

grafik

File: web/src/app/modules/navigation/modules/project-explorer/components/project-explorer.component.html [hunk 1]
```diff
@@ -1,48 +1,52 @@
...
-                    <icon-selector [model]="element" expanded="false"></icon-selector>{{element['extId'] ? element['extId'] + ': ' : ''}}{{element.name}}
-                </a>
-            </td>
-        </tr>
-    </tbody>
-</table>
+<div class="input-group">
+    <input #searchBox class="form-control" type="search" (keyup)="search(searchBox.value)" />
+    <button *ngIf="searchBox.value"
+        (click)="searchBox.value = null" title="{{'delete' | translate}}"
```
Comment (junkerm) [inline web/src/app/modules/navigation/modules/project-explorer/components/project-explorer.component.html]: please make this a code-behind function (e.g. "clearSearch")

Comment (bliemelh) [reply web/src/app/modules/navigation/modules/project-explorer/components/project-explorer.component.html]: Not quite sure what I did there now, but it works. Please tell me, if you know a nicer way to do this.

**Comment by junkerm** [comment]
looks good, nevertheless I added @sebeder as reviewer to check if this is the right way to go for the ui

**Review by sebeder — APPROVED** [review]
Looks good!

**Pull request merged**